test-integration/src/org/omegat/core/data/TestTeamIntegration.java

```diff
@@ -111,11 +118,14 @@
 
     private TestTeamIntegration() {
     }
+    private final static String regex = "http(s)?://(?<username>.+?)(:(?<password>.+?))?@.+";
+    private final static Pattern URL_PATTERN = Pattern.compile(regex);
 
     static final String DIR = "/tmp/teamtest";
-    static final String REPO = System.getProperty("omegat.test.repo", "git@github.com:alex73/trans.git");
-    // static final String REPO = "svn+ssh://alex73@svn.code.sf.net/p/mappy/test/";
-    // static final String REPO = "https://github.com/alex73/trans/trunk/";
+    static final List<String> REPO = new ArrayList<>();
+    static final String MAP_REPO = System.getProperty("omegat.test.maprepo", null);
+    static final String MAP_REPO_TYPE = System.getProperty("omegat.test.maptype", "http");
```

Comment:
It is better to change to `omegat.test.map.repo` and `omegat.test.map.type` as property names.